workflows/riboseq/main.nf

```diff
@@ -316,6 +319,51 @@ workflow RIBOSEQ {
         ch_versions = ch_versions.mix(RIBOWALTZ.out.versions)
     }
 
+    //
+    // Get P-sites with plastid
+    //
+
+    // Keep only riboseq genome BAMs for plastid
+    ch_genome_bam
```

Comment:
You don't need to re-define `ch_genome_bam`, you can use it from the earlier definition